Assets/HoloToolkit/Utilities/Scripts/WorldAnchorManager.cs

```diff
@@ -136,6 +137,40 @@ public void RemoveAnchor(GameObject gameObjectToUnanchor)
         }
 
         /// <summary>
+        /// Removes all anchors from the scene and deletes them from the anchor store.
+        /// </summary>
+        public void RemoveAllAnchors()
+        {
+            SpatialMappingManager spatialMappingManager = SpatialMappingManager.Instance;
```

Comment:
Personal preference in term of readability.
I am fine either way.
Do you want me to inline the "SpatialMappingManager.Instance" usages ?